google-cloud-pubsub/lib/google/cloud/pubsub/subscription.rb

```diff
@@ -513,10 +513,9 @@ def seek snapshot
         # @see https://cloud.google.com/pubsub/docs/reference/rpc/google.iam.v1#iampolicy
         #   google.iam.v1.IAMPolicy
         #
-        # @param [Boolean] force Force the latest policy to be retrieved from
-        #   the Pub/Sub service when `true`. Otherwise the policy will be
-        #   memoized to reduce the number of API calls made to the Pub/Sub
-        #   service. The default is `false`.
+        # @param [Boolean] force [Deprecated] Force the latest policy to be
```

Comment:
Since Pub/Sub isn't 1.0 yet, I think we should remove the option. We can still make breaking changes before we hit 1.0.

I would use these deprecation descriptions for Storage though.